docs/api-reference/advanced/exists.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Exists
+---
+
+## Basic Usage
+
+Matches all documents with a non-null value in the specified field. All matched documents get a BM25 score of `1.0`.
```

Comment:
This is due to my ignorance, but should we document the BM25 score range? If normalized it would be 0-1 but if not it's apparently 1-1000. Which one do we use? I realize I don't know.

Reply:
They don’t. 

And the only way to normalize is if you have the max score of the matching set.